components/OgImage/BlogPost.vue

```
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+defineProps<{
+  title?: string
+  description?: string
+  author?: string
+  date?: string
+  siteName?: string
+}>()
+
+function formatDate(dateString?: string) {
```

ESLint

[error] 'formatDate' is defined but never used.

(no-unused-vars)